components/status_dropdown/status_dropdown.tsx

```diff
@@ -120,32 +125,33 @@ export default class StatusDropdown extends React.PureComponent {
             </FormattedMessage>
         );
     }
-    handleClearStatus = (e) => {
+    handleClearStatus = (e: React.MouseEvent<HTMLButtonElement>): void => {
         e.stopPropagation();
         e.preventDefault();
         this.props.actions.unsetCustomStatus();
     };
 
-    onToggle = (open) => this.props.actions.setStatusDropdown(open);
+    onToggle = (open: boolean): void => {
+        this.props.actions.setStatusDropdown(open);
+    }
 
-    renderCustomStatus = () => {
+    renderCustomStatus = (): ReactNode => {
         if (!this.props.isCustomStatusEnabled) {
             return null;
         }
         const customStatus = this.props.customStatus;
-        const isStatusSet = customStatus && (customStatus.text || customStatus.emoji);
+        const isStatusSet = customStatus && (customStatus.text.trim().length !== 0 || customStatus.emoji.trim().length !== 0);
```

Comment:
Is it possible that the strings will contain only blank spaces?
I'm just wary of changing logic here unnecessarily.

If it's a type issue you're running into, you can cast strings as a Boolean here.

Reply:
Done @devinbinnie